lib/factories/ppt/slides.js

```diff
@@ -315,6 +315,54 @@ class SlideFactory {
 
         return newGraphicFrameBlock;
     }
+
+    setBackgroundColor(slide, color) {
+        let slideKey = `ppt/slides/${slide.name}.xml`;
+        let slideContent = this.content[slideKey]['p:sld']['p:cSld'][0];
+        let existingNodes = [];
+
+        if (slideContent['p:bg']) {
+            if (!slideContent['p:bg'][0]['p:bgPr']) {
```

Comment:
can we make this an explicit test on undefined?

Reply:
Done.